scipy/sparse/linalg/tests/test_pydata_sparse.py

```diff
@@ -176,9 +176,9 @@ def test_splu(matrices):
     Pc = sparse_cls(sp.csc_matrix((np.ones(n), (np.arange(n), lu.perm_c))))
     A2 = Pr.T @ lu.L @ lu.U @ Pc.T
 
-    assert_allclose(A2.toarray(), A_sparse.toarray())
+    assert_allclose(A2.todense(), A_sparse.todense())
```

Comment:
Why change from `.toarray()` to `.todense()` here?

Reply:
I think `.toarray()` is incorrect. `pydata/sparse` object don't have this function, they provide `.todense()` instead.
`.toarray()` is `scipy.sparse`'s method, so I think it's incorrect here, considering that this test suite isn't even run in the CI in the main branch.